app/src/main/res/values-ar/readers.xml

```diff
@@ -15,7 +15,7 @@
     <string name="qari_hani_rifai">هاني الرفاعي</string>
     <string name="qari_husary_mujawwad">الحصري مجود</string>
     <string name="qari_hudhayfi">الحذيفي</string>
-    <string name="qari_muaiqly">ماهر المعيلقي</string>
+    <string name="qari_muaiqly">ماهر_المعيقلي</string>
```

Comment:
Sorry, underscore was unintentional. I only wanted to correct the sequence
of letters.
On Jan 14, 2016 00:45, "Ahmed El-Helw" <notifications@github.com> wrote:

> In app/src/main/res/values-ar/readers.xml
> <https://github.com/quran/quran_android/pull/616#discussion_r49640472>:
>
> > @@ -15,7 +15,7 @@
> >      <string name="qari_hani_rifai">هاني الرفاعي</string>
> >      <string name="qari_husary_mujawwad">الحصري مجود</string>
> >      <string name="qari_hudhayfi">الحذيفي</string>
> > -    <string name="qari_muaiqly">ماهر المعيلقي</string>
> > +    <string name="qari_muaiqly">ماهر_المعيقلي</string>
>
> why did you add an _ here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran_android/pull/616/files#r49640472>.
>